doc/holochain_101/src/zome/callback_functions.md

```diff
@@ -1,25 +1,24 @@
-# Lifecycle functions
+# Callback functions
 
 ## Overview
 
-A lifecycle function is implemented in the zome language and called by
-Holochain.
+A callback function is implemented in the zome language and called by Holochain.
 
 Contrast this to a zome API function that is implemented by Holochain and called
 by the zome.
 
-Lifecycle functions are passed some parameters and expected to return one of
-three possible `LifecycleFunctionParams` variants:
+Callback functions are passed some parameters and expected to return one of
+three possible `CallbackParams` variants:
 
 - `Pass`: The lifecycle function has executed/validated successfully
```

Comment:
`lifecycle` -> `callback`

Reply:
you can just edit this in github UI... @sphinxc0re